Main.cpp

```diff
@@ -0,0 +1,127 @@
+/* 
+ @file      Main.cpp
+ @brief     creates geometry and materials to run a Monte Carlo simulation
+ @author    Luke Eure
+ @date      January 9 2016
+*/
+
+#include <iostream>
+#include <vector>
+#include <time.h>
+#include <math.h>
+#include <stdlib.h>
+
+#include "Material.h"
+#include "Boundaries.h"
+#include "Distributions.h"
+#include "Tally.h"
+#include "Mesh.h"
+#include "Monte_carlo.h"
+#include "Plotter.h"
+#include "Neutron.h"
+
+int main() {
+    
+    srand(time(NULL));
+
+    // create geometry
+    Boundaries test_boundary;
+    test_boundary.setSurfaceCoord(0, 0, -2.0);
+    test_boundary.setSurfaceCoord(0, 1, 2.0);
+    test_boundary.setSurfaceCoord(1, 0, -2.0);
+    test_boundary.setSurfaceCoord(1, 1, 2.0);
+    test_boundary.setSurfaceCoord(2, 0, -2.0);
+    test_boundary.setSurfaceCoord(2, 1, 2.0);
+    test_boundary.setSurfaceType(0, 0, VACUUM);
+    test_boundary.setSurfaceType(0, 1, VACUUM);
+    test_boundary.setSurfaceType(1, 0, VACUUM);
+    test_boundary.setSurfaceType(1, 1, VACUUM);
+    test_boundary.setSurfaceType(2, 0, VACUUM);
+    test_boundary.setSurfaceType(2, 1, VACUUM);
+
+    const int num_groups = 2;
+
+    // fuel cross sections
+    std::vector <double> fuel_sigma_f;
+    fuel_sigma_f.push_back(1.0/480.0);
+    fuel_sigma_f.push_back(1.0/16.0);
+    std::vector <double> fuel_chi;
+    fuel_chi.push_back(1.0);
+    fuel_chi.push_back(0.0);
+    std::vector <double> fuel_sigma_t;
+    fuel_sigma_t.push_back(2.0/9.0);
+    fuel_sigma_t.push_back(5.0/6.0);
```

Comment:
Here we again know the length of ``fuel_sigma_f``, ``fuel_chi``, and ``fuel_sigma_t`` so let's initialize them of size ``num_groups``